### PR TITLE
Adding in missing _RESERVED values in HandshakeType

### PR DESCRIPTION
**Pull request by seanturner**
Apologies if I'm doing this wrong.

I think these values should be included in the document.  They were specified in earlier versions and should be included but marked as reserved.

### CODE REVIEW

File: rfc8446.xml [hunk 1]
```diff
@@ -830,6 +830,9 @@ processed and transmitted as specified by the current active connection state.</
        certificate_request(13),
        certificate_verify(15),
        finished(20),
+       certificate_URL_RESEREVED(21),
```
Comment (martinthomson) [inline rfc8446.xml]: the registry has "url" in lowercase

**Review by martinthomson — APPROVED** [review]
I checked the IANA registry and these are OK (with a nit).

The net result of these changes is a little surprising (it marks things "RESERVED" that are still in wide use), but I can live with that.

File: rfc8446.xml (hunk 1 above)
Comment (jrchamp) [reply rfc8446.xml]: Also, an extra 'E' snuck in to RESER**E**VED.

**Comment by seanturner** [comment]
@martinthomson I was going with the thought what's in the intro of Appendix B:

    Values listed as _RESERVED were used in previous
    versions of TLS and are listed here for completeness.

**Comment by jrchamp** [comment]
Do the other ones that disappear in TLS 1.3 need to be listed also?
https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#table-tls-parameters-7

**Comment by seanturner** [comment]
I'm afraid I'm doing something wrong here because the other values 0, 3, 6, 12, 14, and 16 are already marked as reserved in the draft.

**Review by kaduk — CHANGES_REQUESTED** [review]
This is something of an artifact of editing the XML instead of the Markdown source -- the Markdown-to-XML processing put the full protocol bits in appendices and stripped the ```_RESERVED``` values for the main text.  But that means that the enums appear twice in the XML, so you want to be editing the "handshake-protocol-1" section and not this one.

**Pull request closed without merging**